test/tests/preconditioners/fsp/sbp_test.i

```diff
@@ -0,0 +1,122 @@
+[Mesh]
+  file = square.e
+#  init_unif_refine = 6
+[]
+
+[Variables]
+  active = 'u v'
+
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./v]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  active = 'diff_u conv_v diff_v'
+
+  [./diff_u]
+    type = Diffusion
+    variable = u
+  [../]
+
+  [./conv_v]
+    type = CoupledForce
+    variable = v
+    v = u
+  [../]
+
+  [./diff_v]
+    type = Diffusion
+    variable = v
+  [../]
+[]
+
+[BCs]
+  active = 'left_u right_u left_v'
+
+  [./left_u]
+    type = DirichletBC
+    variable = u
+    boundary = 1
+    value = 0
+  [../]
+
+  [./right_u]
+    type = DirichletBC
+    variable = u
+    boundary = 2
+    value = 100
+  [../]
+
+  [./left_v]
+    type = DirichletBC
+    variable = v
+    boundary = 1
+    value = 0
+  [../]
+
+  [./right_v]
+    type = DirichletBC
+    variable = v
+    boundary = 2
+    value = 0
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+
+  # This is setup automatically in MOOSE (SetupPBPAction.C)
+  # petsc_options = '-snes_mf_operator'
+  # petsc_options_iname = '-pc_type'
+  # petsc_options_value =  'asm'
+[]
+
+[Preconditioning]
+  active = 'FSP'
+
+  [./PBP]
+    type = PBP
+    solve_order = 'u v'
+    preconditioner  = 'LU LU'
+    off_diag_row    = 'v'
+    off_diag_column = 'u'
+
+    petsc_options = ''  # Test petsc options in PBP block
+  [../]
+
+  [./FSP]
```

Comment:
Add a comment here describing the linear algebra of this preconditioner so that people know how it works...